docs/source/managed-federation/client-awareness.mdx

```diff
@@ -1,9 +1,12 @@
 ---
-title: Client awareness
-description: Configure client awareness in Apollo Router
+title: Client Awareness
+subtitle: Configure client awareness in the GraphOS Router
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)